src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Shapes/Basics_Shapes_Tests.cs

```diff
@@ -155,8 +155,8 @@ public void ValidateShape(string shapeName, PixelTolerance? tolerance = null)
 						}
 
 						using (var actual = (Bitmap)testResult)
-						{ 
-							var scale = _app.GetDisplayScreenScaling();
+						{
+							var scale = 1d;
```

Comment:
Do you recall why you needed to skip the actual scaling ?

Reply:
If I remember correctly RenderTargetBitmap using logical pixels